src/__tests__/component/Settings.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import Enzyme, { shallow } from 'enzyme';
+import EnzymeAdapter from 'enzyme-adapter-react-16';
+import Settings from '../../components/Profile/Settings/Settings';
+
+Enzyme.configure({ adapter: new EnzymeAdapter() });
+
+test('renders without an error', () => {
+  const wrapper = shallow(<Settings />);
+  expect(wrapper.find('li').length).toBe(2);
```

Comment:
@MBenedicte  If someone adds another `<li>` I think test can throw an unexpected error because you expect lists to explicitly equal to 2.